priority-queue/PriorityQueueConsumerHigh/Properties/serviceDependencies.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "appInsights1": {
+      "type": "appInsights"
+    },
+    "storage1": {
+      "type": "storage",
+      "connectionId": "AzureWebJobsStorage"
+    }
```

Comment:
This part in this file doesn't exist in the "Low" version -- is there a reason it exists in one and doesn't exist in the other?

Reply:
This file is present when you create an Azure functions project from scratch. So I decided to keep it, containing the default values:

{
  "dependencies": {
    "appInsights1": {
      "type": "appInsights"
    },
    "storage1": {
      "type": "storage",
      "connectionId": "AzureWebJobsStorage"
    }
  }
}

Reply:
Thanks for syncing them up.